pkg/controller/provider/controller.go

```diff
@@ -184,6 +201,55 @@ func (r Reconciler) Reconcile(ctx context.Context, request reconcile.Request) (r
 		}
 	}
 
+	//if the provider added is ova we need to lunch a new pod
```

Comment:
we should post a deployment here rather than a pod and ideally set the provider as its owner